frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java

```diff
@@ -84,9 +84,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
   private void doFastForwardCurrentBranch(Project project,
       GitRepository gitRepository,
       INonRootManagedBranchSnapshot targetBranch) {
-
```

Comment:
Minor typos I've spotted, while you're here:
`build.gradle:14: s/amd/and`
`.circleci/config.yml:21: s/got/go`